modules/nf-core/sylphtax/taxprof/main.nf

```diff
@@ -0,0 +1,50 @@
+
+process SYLPHTAX_TAXPROF {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/sylph-tax:1.1.2--pyhdfd78af_0':
+        'biocontainers/sylph-tax:1.1.2--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(sylph_results)
+    path taxonomy
+
+    output:
+    tuple val(meta), path("*.sylphmpa")         , emit: taxprof_output
+    path "versions.yml"                         , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    """
+    sylph-tax \\
+        taxprof \\
+        $sylph_results \\
+        $args \\
+        -o ${prefix}.sylphmpa \\
```

Comment:
Suffix `sylphmpa` is fixed on purpose for downstream tools?

Reply:
This is the output suffix or what do you mean with the downstream tools?

Reply:
My thinking was about whether it's required fix this suffix here as a string or if it removes flexibility to name the output file. You know the tool better than me, so if that's the common suffix, leave it as is :+1:

Reply:
This is the common suffix so I leave it as it is.